Magicnumbers/Magicnumbers.cpp

```diff
@@ -63,6 +57,5 @@ int main() {
 	exitchoice();
```

Comment:
Yeah, but thats the one part of the code I didn't change. This PR is about adding legacy version back with level saving. So I feel it would be good to remove it in a different pull request